single-click-setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
```

Comment:
Please remove this file. Not needed on Flathub.

Reply:
Removed it. I'll keep it local next time